src/XML/ds/Transform.php

```diff
@@ -83,17 +83,6 @@ private function setElements(array $elements): void
     }
 
 
-    /**
-     * Test if an object, at the state it's in, would produce an empty XML-element
-     *
-     * @return bool
-     */
-    public function isEmptyElement(): bool
```

Comment:
This was a mistake; Transform cannot be empty

Reply:
Yeah, I noticed it but assumed that it was just a mistake :-)